internal/experimental/registry/reference.go

```diff
@@ -100,7 +100,7 @@ func (ref *Reference) validateRepo() error {
 	}
 	// Makes sure the repo results in a parsable URL (similar to what is done
 	// with containerd reference parsing)
-	_, err := url.Parse(ref.Repo)
+	_, err := url.Parse("//" + ref.Repo)
```

Comment:
can you add a comment here explaining the fix? That way the next person reading this can better understand what the `//` is supposed to do. Thanks.

Reply:
PTAL